CHANGELOG.rst

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ==========
 
+- #17: Like with ``virtualenv``, venvs are always created with a
+  late version of pip, setuptools, and wheel.
```

Comment:
«a late version»? does that mean: the latest version?

Reply:
I'm of the sense that sometimes the latest version is not installed. In fact, just now I created a venv using tox-venv with this patch, and it got pip 19.0.3 not 19.1:

```
pip-run master $ tox -r --notest                                                                                                                                               
python recreate: /Users/jaraco/code/main/pip-run/.tox/python
python installdeps: setuptools>=31.0.1
python develop-inst: /Users/jaraco/code/main/pip-run
python installed: atomicwrites==1.3.0,attrs==19.1.0,docutils==0.14,entrypoints==0.3,flake8==3.7.7,importlib-metadata==0.9,mccabe==0.6.1,more-itertools==7.0.0,-e git+gh://jaraco/pip-run@b92009576f96c4e02cf7f22db6db393ee18d3fbe#egg=pip_run,pluggy==0.9.0,py==1.8.0,pycodestyle==2.5.0,pyflakes==2.1.1,pytest==4.4.1,pytest-checkdocs==1.2.0,pytest-flake8==1.0.4,six==1.12.0,zipp==0.3.3
___________________________________________________________________________________ summary ___________________________________________________________________________________
  python: skipped tests
  congratulations :)

pip-run master $ .tox/python/bin/pip --version                                                                                                                                 
pip 19.0.3 from /Users/jaraco/code/main/pip-run/.tox/python/lib/python3.7/site-packages/pip (python 3.7)
pip-run master $ which tox                                                                                                                                                     
/Users/jaraco/.local/bin/tox
pip-run master $ ~/.local/pipx/venvs/tox/bin/pip show tox-venv                                                                                                                 
Name: tox-venv
Version: 0.4.0+pr25
Summary: Use Python 3 venvs for Python 3 tox testenvs
Home-page: https://github.com/tox-dev/tox-venv
Author: Ryan P Kilby
Author-email: kilbyr@gmail.com
License: BSD
Location: /Users/jaraco/.local/pipx/venvs/tox/lib/python3.7/site-packages
Requires: tox
Required-by: 
```

So I don't think it's always the latest version.

Weirdly, I end up with a different version when using virtualenv:

```
pip-run master $ ~/.local/pipx/venvs/tox/bin/python -m virtualenv venv                                                                                                         
Using base prefix '/Library/Frameworks/Python.framework/Versions/3.7'
New python executable in /Users/jaraco/code/main/pip-run/venv/bin/python
Installing setuptools, pip, wheel...
done.
pip-run master $ venv/bin/pip --version                                                                                                                                        
pip 19.1 from /Users/jaraco/code/main/pip-run/venv/lib/python3.7/site-packages/pip (python 3.7)
```

This finding is indeed disconcerting.

Reply:
I think to get _equivalent_ behavior (the intention of this PR), it may be necessary to invoke the [virtualenv_support_dirs context](https://github.com/pypa/virtualenv/blob/debbaa4ff2712167006f21d978267968c2191d2b/virtualenv.py#L474).

Reply:
Hmm. Now that I look at it again, I wonder if this patch is having any effect at all. At one point, I'm certain it was. But right now, I'm seeing the same version of pip appearing from vanilla 
`python -m venv` as from "tox+tox-venv".

Reply:
I see now that [virtualenv_support_dirs is the default](https://github.com/pypa/virtualenv/blob/debbaa4ff2712167006f21d978267968c2191d2b/virtualenv.py#L968). Probably the difference I'm seeing in behavior is the fact that [download defaults to True](https://github.com/pypa/virtualenv/blob/debbaa4ff2712167006f21d978267968c2191d2b/virtualenv.py#L692) for the command-line invocation but [defaults to False](https://github.com/pypa/virtualenv/blob/debbaa4ff2712167006f21d978267968c2191d2b/virtualenv.py#L966) for the API invocation.

Reply:
In this latest commit (7c0f2b5), I pass the `download=True` to the `install_wheel`, which causes the install to download the latest versions, same as virtualenv does when invoked by tox without venv, and so I've updated the changelog accordingly.